modules/spo-installing-cli.adoc

```diff
@@ -8,7 +8,7 @@
 
 .Prerequisites
 
-* You must have `admin` privileges.
+* You must have `cluster-admin` privileges.
```

Comment:
In several other similar revisions, you have "You have access to the web console as a user with `cluster-admin` privileges." Should the wording here say that instead?